src/components/publishing/sections/text_container.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import Text from "./text"
+
+interface TextContainerProps extends React.HTMLProps<HTMLDivElement> {
+  layout: "classic" | "feature" | "standard"
+  postscript?: boolean
+  html?: string
+}
+
+const TextContainer: React.SFC<TextContainerProps> = props => {
+  const { html, layout, postscript } = props
+  const child = html ? <div dangerouslySetInnerHTML={{ __html: html }} /> : props.children
+  return (
+    <Text layout={layout} postscript={postscript}>
+      {child}
+    </Text>
+  )
+}
+export default TextContainer
```

Comment:
This is a bit of a nitpick, but could we name this `Text` and make the styled div either in the same file as this or separate and call it something like `StyledText`? Unless I'm missing a good reason not to.

I only say this because our components at the top-most level are consistently named like: `Embed`, `Video`, `Header`, etc. Not necessary but just a thought.